dataland-frontend/src/components/pages/SearchCompaniesForFrameworkData.vue

```diff
@@ -12,35 +12,50 @@
           <FrameworkDataSearchBar
             v-model="currentSearchBarInput"
             ref="frameworkDataSearchBar"
-            :searchBarName="searchBarName"
-            :frameworksToFilterFor="currentFilteredFrameworks"
+            :filter="currentCombinedFilter"
+            :search-bar-name="searchBarName"
+            :enable-full-search="true"
             @companies-received="handleCompanyQuery"
-            @rendered="handleFrameworkDataSearchBarRender"
-            class="pl-4"
+            @search-confirmed="handleSearchConfirmed"
           />
-
+          <div class="flex">
```

Comment:
We have to change the scroll behaviour a litte bit.
The switch between the top search bar and the toggle search bar must come at a greater scroll distance than now, because otherwise you see the Filter twice at certain scroll distances.
Maybe this will be fixed when we use only one FilterContainer and change its class conditionally.